From dce310e34f536f188beec35e32a3e1632e0949d3 Mon Sep 17 00:00:00 2001
From: Szymon Kaliski <kaliskiszymon@gmail.com>
Date: Tue, 21 Feb 2017 11:03:04 +0100
Subject: [PATCH] handle accesory window policy

this allows kwm to tile hs console for example, more info: https://github.com/koekeishiya/kwm/issues/565#issuecomment-277507891
---
 axlib/sharedworkspace.mm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/axlib/sharedworkspace.mm b/axlib/sharedworkspace.mm
index 08126af..af1380a 100644
--- a/axlib/sharedworkspace.mm
+++ b/axlib/sharedworkspace.mm
@@ -22,7 +22,8 @@ shared_ws_map SharedWorkspaceRunningApplications()
     shared_ws_map List;
     for(NSRunningApplication *Application in [[NSWorkspace sharedWorkspace] runningApplications])
     {
-        if([Application activationPolicy] == NSApplicationActivationPolicyRegular)
+        // if([Application activationPolicy] == NSApplicationActivationPolicyRegular)
+        if([Application activationPolicy] == NSApplicationActivationPolicyRegular || [Application activationPolicy] == NSApplicationActivationPolicyAccessory)
         {
             pid_t PID = Application.processIdentifier;
             std::string Name = "[Unknown]";
-- 
2.12.0

